### ISSUE TITLE
"About Cookies" slideover does not return focus to About Cookies button when dismissed

### ISSUE THREAD

**Issue by MrMcFortna** [app-illinois/Design-Resources#189]
When the "About Cookies" slideover is dismissed, its focus does not return to the About Cookies button that triggered the slideover. Returning focus to the triggering button is standard behavior for popups/modals. Ideally, this behavior would be baked into the slideover widget - whenever a slideover is activated via user interaction, dismissing the slideover would return focus to the triggering button. In the short term, since it appears there's only one button that triggers this slideover, you could just manually return focus to that button without changing the slideover widget's behavior more broadly. Note that it's necessary to implement this focus management no matter how the slideover is dismissed: clicking the X button, clicking the Close button, pressing `<esc>`, etc.

**Comment by edthedev** [comment]
@MrMcFortna Thank you for the bug report. We will get this fixed.

**Comment by edthedev** [comment]
@fleming7 I might need to beg for your help on this one. 

I see that the slide-over component has focus management capabilities. 

But the way we use it in the cookie banner doesn't result in focus returning as expected.

We're doing a bunch of exciting dynamic interesting (half-crazy, worrisome, weird) things to make the banner work from a single JavaScript import, which could be related. 

If needed, we can probably hack a work-around, but I have a feeling I might be missing something important, and would appreciate a few minutes of your insights, when you're available.

**Comment by edthedev** [comment]
This may have been addressed by fixes #182 and #193, but we will re-test before we close this. Thanks @fleming7 for the collaboration on this!

**Comment by edthedev** [comment]
Another change is in progress to address this in #194
